### PR TITLE
Fix ggplot2 tests: modify validators

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [x] - This PR does something else (explain below)

This PR fixes a test, which fails due to lint tool-list.

### CODE REVIEW

File: tools/ggplot2/ggplot2_pca.xml [hunk 1]
```diff
@@ -249,11 +249,19 @@ plot_out <- plot_out +
...
+                        <sanitizer invalid_char="">
+                            <valid initial="string.letters,string.digits">
+                                <add value=":" />
+                                <add value="-" />
+                                <add value=" " />
+                                <add value="_" />
+                            </valid>
+                        </sanitizer>
+                        <validator type="regex">[0-9a-zA-Z:-_ ]+</validator>
+                    </param>                    </param>
```
Comment (bernt-matthias) [inline tools/ggplot2/ggplot2_pca.xml:261]: There is 2x `</param>`. 

Maybe add a mesage to the validator. Is the combination of sanitizerr + vaidator useful? One of them might be enough.

Comment (gallardoalba) [reply tools/ggplot2/ggplot2_pca.xml:261]: Thanks, I didn't realize that this information was included in the report.

Comment (gallardoalba) [reply tools/ggplot2/ggplot2_pca.xml]: Bjorn explained to me that they can be complementary at some point, in order to provide more security; so I'm not sure in which cases I can use just one of them.

Comment (bernt-matthias) [reply tools/ggplot2/ggplot2_pca.xml]: Yep, my mistake.

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**